tendrl/node_agent/monitoring/collectd/collectors/gluster/heavy_weight/tendrl_glusterfs_profile_info.py

```diff
@@ -70,150 +68,148 @@
 ]
 
 
-def _parseVolumeProfileInfo(tree, nfs=False):
-    bricks = []
-    if nfs:
-        brickKey = 'nfs'
-        bricksKey = 'nfsServers'
-    else:
-        brickKey = 'brick'
-        bricksKey = 'bricks'
-    for brick in tree.findall('volProfile/brick'):
-        fopCumulative = []
-        blkCumulative = []
-        fopInterval = []
-        blkInterval = []
-        brickName = brick.find('brickName').text
-        for block in brick.findall('cumulativeStats/blockStats/block'):
-            blkCumulative.append(
-                {
-                    'size': block.find('size').text,
-                    'read': block.find('reads').text,
-                    'write': block.find('writes').text
-                }
-            )
-        for fop in brick.findall('cumulativeStats/fopStats/fop'):
-            gevent.sleep(0.1)
-            fopCumulative.append(
-                {
-                    'name': fop.find('name').text,
-                    'hits': fop.find('hits').text,
-                    'latencyAvg': fop.find('avgLatency').text,
-                    'latencyMin': fop.find('minLatency').text,
-                    'latencyMax': fop.find('maxLatency').text
-                }
-            )
-        for block in brick.findall('intervalStats/blockStats/block'):
-            gevent.sleep(0.1)
-            blkInterval.append(
-                {
-                    'size': block.find('size').text,
-                    'read': block.find('reads').text,
-                    'write': block.find('writes').text
-                }
-            )
-        for fop in brick.findall('intervalStats/fopStats/fop'):
-            gevent.sleep(0.1)
-            fopInterval.append(
+class TendrlHealInfoAndProfileInfoPlugin(
+    TendrlGlusterfsMonitoringBase
+):
+    def __init__(self):
+        self.provisioner_only_plugin = True
+        TendrlGlusterfsMonitoringBase.__init__(self)
+
+    def _parseVolumeProfileInfo(self, tree, nfs=False):
+        bricks = []
+        if nfs:
+            brickKey = 'nfs'
+            bricksKey = 'nfsServers'
```

Comment:
These variable names are similar and confusing. Use some more meaningful name

Reply:
Its oVirt VDSM parsing logic.
No net new code from existing plugin state except they have been converted into class extending our plugin framework within collectd